src/features/shared-link-settings-modal/__tests__/ExpirationSection.test.js

```diff
@@ -11,11 +11,18 @@ describe('features/shared-link-settings-modal/ExpirationSection', () => {
     const isExpirationEnabled = true;
     const onCheckboxChange = sandbox.stub();
     const onExpirationDateChange = sandbox.stub();
+    const dateDisplayFormat = {
+        day: 'numeric',
+        month: 'long',
+        year: 'numeric',
+        timeZone: 'Europe/Warsaw',
+    };
 
     const getWrapper = (props = {}) =>
         shallow(
             <ExpirationSection
                 canChangeExpiration={canChangeExpiration}
+                dateDisplayFormat={dateDisplayFormat}
```

Comment:
Since this is now added to the wrapper, I think we need one more unit test to account that this is also defaulted if `dateDisplayFormat` is not provided